src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ExplicitInitializationCheck.java

```diff
@@ -67,15 +67,19 @@
  * </p>
  * <pre>
  * public class Test {
```

Comment:
please add example of code below default config to show users how it works by default.
The same code should be used in config with property to let user see difference in behavior.